runtime/near-vm-runner/src/prepare/prepare_v2.rs

```diff
@@ -1,58 +1,326 @@
 use crate::internal::VMKind;
+use finite_wasm::wasmparser as wp;
 use near_vm_errors::PrepareError;
 use near_vm_logic::VMConfig;
+use wasm_encoder::{Encode, Section, SectionId};
+
+pub const WASM_FEATURES: wp::WasmFeatures = wp::WasmFeatures {
+    mutable_global: true,
+    reference_types: super::WASM_FEATURES.reference_types,
+    multi_value: super::WASM_FEATURES.multi_value,
+    bulk_memory: super::WASM_FEATURES.bulk_memory,
+    simd: super::WASM_FEATURES.simd,
+    relaxed_simd: super::WASM_FEATURES.simd,
```

Comment:
Fair, I effectively took a shortcut here since simd is disabled right now and it is going to be the case for foreseeable future (given that our codegen does not support it either way.) Happy to just set it to `false` instead too.